specifications/sequence-diagrams/README.md

```diff
@@ -0,0 +1,8 @@
+# Payment Channel Sequence Diagrams
+
+## How to view
+These sequence diagrams were created using the [mermaid diagram live editor](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtXG4gICAgcGFydGljaXBhbnQgSSBhcyBJbml0aWF0b3JcbiAgICBwYXJ0aWNpcGFudCBSIGFzIFJlc3BvbmRlclxuICAgIFxuICAgIE5vdGUgb3ZlciBJLFI6IDEuIFNldHVwXG4gICAgTm90ZSByaWdodCBvZiBSOiBpID0gMCwgZSA9IDBcbiAgICBJLS0-Pkk6IGNyZWF0ZSBhbmQgZGVwb3NpdCBmdW5kcyB0byBFSVxuICAgIFItLT4-UjogY3JlYXRlIGFuZCBkZXBvc2l0cyBmdW5kcyB0byBFUlxuICAgIEktPj5SOiBDcmVhdGUgZm9ybWF0aW9uIHRyYW5zYWN0aW9uIEZcbiAgICBOb3RlIHJpZ2h0IG9mIFI6IGkgPSAxXG4gICAgSS0-PlI6IENyZWF0ZSBhbmQgc2lnbiBDXzFcbiAgICBSLS0-Pkk6IHNpZ24gQ18xLCBzZW5kIGJhY2tcbiAgICBJLT4-UjogQ3JlYXRlIGFuZCBzaWduIERfMVxuICAgIFItLT4-STogc2lnbiBEXzEsIHNlbmQgYmFja1xuICAgIEktPj5SOiBTaWduIEZcbiAgICBSLT4-UjogU2lnbiBGLCBzdWJtaXRcblxuICAgIE5vdGUgb3ZlciBJLFI6IDIuIFVwZGF0ZVxuICAgIE5vdGUgcmlnaHQgb2YgUjogaT0yXG4gICAgSS0-PlI6IENyZWF0ZSBhbmQgc2lnbiBDXzJcbiAgICBSLS0-Pkk6IFNpZ24gQ18yLCBzZW5kIGJhY2tcbiAgICBJLT4-UjogQ3JlYXRlIGFuZCBzaWduIERfMlxuICAgIFItLT4-STogU2lnbiBELTIsIHNlbmQgYmFja1xuXG4gICAgTm90ZSBvdmVyIEksUjogMy4gQ2xvc2luZ1xuICAgIGFsdCBDb29yZGluYXRlZCBDbG9zZVxuICAgICAgICBOb3RlIHJpZ2h0IG9mIFI6IGUgPSAyXG4gICAgICAgIEktPj5JOiBzdWJtaXQgRF8yXG4gICAgICAgIEktPj5SOiBtb2RpZnkgQ18yIHRvIGJlIGFibGUgdG8gc3VibWl0IGltbWVkaWF0ZWx5LCBzaWduXG4gICAgICAgIFItPj5SOiByZS1zaWduIENfMiwgc3VibWl0XG4gICAgZWxzZSBVbmNvb3JkaW5hdGVkIENsb3NlXG4gICAgICAgIEktPj5JOiBzdWJtaXQgRF8xXG4gICAgICAgIFItPj5SOiBzdWJtaXQgRF8yLCB3YWl0IG9ic2VydmF0aW9uIHBlcmlvZCBPIHRvIHBhc3NcbiAgICAgICAgUi0-PlI6IHN1Ym1pdCBDXzJcbiAgICBlbmQiLCJtZXJtYWlkIjp7fSwidXBkYXRlRWRpdG9yIjpmYWxzZX0) which uses this [syntax](https://mermaid-js.github.io/mermaid/#/sequenceDiagram). To view the sequence diagram copy the text into the `code` section.
```

Comment:
💡 Suggest simplifying this link since this link appears intended to take the reader to the live editor ready for them to paste in the diagrams below, but the link is taking them to the live editor pre-populated with a diagram that will likely become out-of-date over time.
```suggestion
These sequence diagrams were created using the [mermaid diagram live editor](https://mermaid-js.github.io/mermaid-live-editor) which uses this [syntax](https://mermaid-js.github.io/mermaid/#/sequenceDiagram). To view the sequence diagrams below copy the text into the `code` section of the live editor.
```